.gitignore

```diff
@@ -0,0 +1,36 @@
+*.rbc
+capybara-*.html
+.rspec
+/log
+/tmp
+/db/*.sqlite3
+/db/*.sqlite3-journal
+/public/system
+/coverage/
+/spec/tmp
+**.orig
+rerun.txt
+pickle-email-*.html
+
```

Comment:
So, using the default Rails .gitignore was a bad idea. We should nuke 90% of this shit.